cmake/AddPdfTarget.cmake

```diff
@@ -29,16 +29,16 @@ function (add_pdf_target docname lang entities figures dtd_files)
     file(GLOB xsl_files "${CMAKE_SOURCE_DIR}/xsl/*.xsl")
     file(GLOB gnucash_icon_files "${CMAKE_SOURCE_DIR}/xsl/icons/*")
 
-    set(pdffile "${docname}.pdf")
-
 
     # Prepare ${BUILD_DIR}
     add_custom_command(
         OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-preparedir-trigger"
         COMMAND ${CMAKE_COMMAND} -E make_directory "${BUILD_DIR}/figures"
         COMMAND ${CMAKE_COMMAND} -E make_directory "${BUILD_DIR}/images/callouts"
+        COMMAND ${CMAKE_COMMAND} -E make_directory "${OUTPUT_DIR}"
```

Comment:
Like all the other directories, please use cmake's `file(ADD_DIRECTORY ...)` rather than a custom command. As explained before we don't need to track existence of directories as they don't play in dependency resolution.